Lib/fontmake/instantiator.py

```diff
@@ -517,9 +531,11 @@ def collect_info_masters(
 ) -> List[Tuple[Location, FontMathObject]]:
     """Return master Info objects wrapped by MathInfo."""
     locations_and_masters = []
+
     for source in designspace.sources:
-        if source.layerName is not None:
-            continue  # No font info in source layers.
+        if source.layerName is not None and source.layerName != DEFAULT_LAYER_NAME:
+            if source != designspace.default:
```

Comment:
don't use the name to check if a layer is the default, also it's not relevant here what the name of a source layer is. Here we simply want to skip any non-default sources, so the latter check is sufficient (better to use `is not` in this case): `if source is not designspace.default`.

Reply:
OK fixed.